### PR TITLE
Move function app files to `functions/` directory

### PR DESCRIPTION
**Pull request by nooreldeensalah**

### CODE REVIEW

File: functions/delete/main.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        else:
+            name = req_body.get('name')
+
+    if name:
+        return func.HttpResponse(f"Hello, {name}. This HTTP triggered function executed successfully.")
+    else:
+        return func.HttpResponse(
+             "This HTTP triggered function executed successfully. Pass a name in the query string or in the request body for a personalized response.",
+             status_code=200
+        )
```
Comment (AbdulrhmnGhanem) [inline functions/delete/main.py:25]: Is this necessary? I guess the user knows his name, no need to send it back.

File: functions/update/main.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        else:
+            name = req_body.get('name')
+
+    if name:
+        return func.HttpResponse(f"Hello, {name}. This HTTP triggered function executed successfully.")
+    else:
+        return func.HttpResponse(
+             "This HTTP triggered function executed successfully. Pass a name in the query string or in the request body for a personalized response.",
+             status_code=200
+        )
```
Comment (AbdulrhmnGhanem) [inline functions/update/main.py:25]: Same here

File: functions/delete/main.py (hunk 1 above)
Comment (nooreldeensalah) [reply functions/delete/main.py:25]: Yeah, this is just Microsoft's template for HTTP trigger functions, I haven't implemented if yet.

File: functions/update/main.py (hunk 2 above)
Comment (nooreldeensalah) [reply functions/update/main.py:25]: Yeah, this is just Microsoft's template for HTTP trigger functions, I haven't implemented if yet.

Comment (nooreldeensalah) [reply functions/update/main.py:25]: In a future pull request I'll impement both functions, now I want to test the functions after moving them into a different directory.

**Review by AbdulrhmnGhanem — APPROVED** [review]

**Pull request merged**